### ISSUE TITLE
[1.10.2] Elevators now limited to 8 floors?

### ISSUE THREAD

**Issue by CaptainSisko** [McJty/RFTools#1354]
Since the latest update it appears that Elevators are now limited to 8 floors.

The screen will display more than 8 floors but i am unable to select any floors higher than floor 8.

Is this working as intended or a bug?

**Comment by McJty** [comment]
That is not intended but can you give more details? How is it configured? Screenshots...

**Comment by CaptainSisko** [comment]
Sorry about that, i forgot to add the screenshots.

I have this test setup at the moment
https://puu.sh/xs1KU/41d4ffa01d.png

The buttons are all showing on the screen
https://puu.sh/xs1Lt/9c7deee360.png

The elevator will move to all floors but 9, 10 and 11.

I just changed the display to horizontal and it now goes to floor 9 but the other 2 floors are off the screen so i cant test those.
https://puu.sh/xs1QP/8ab6645075.png

So it appears as though it could be an issue with the right column of the vertical display

The above tests are done using the Creative Screen. I just set up a standard screen with a controller and it has the same issue.
https://puu.sh/xs1ZA/50b0ae5771.png
